src/modules/flow/http-client/http-client.c

```diff
@@ -652,15 +754,14 @@ string_post_process(struct sol_flow_node *node, void *data, uint16_t port, uint1
  * --------------------------------- irange node -----------------------------
  */
 static int
-int_process_token(struct sol_flow_node *node,
-    struct sol_json_token *key, struct sol_json_token *value)
+int_process_json(struct sol_flow_node *node, const struct sol_str_slice slice)
 {
     struct sol_irange irange = SOL_IRANGE_INIT();
     enum sol_json_loop_reason reason;
     struct sol_json_scanner sub_scanner;
     struct sol_json_token sub_key, sub_value, token;
 
-    sol_json_scanner_init(&sub_scanner, value->start, value->end - value->start);
+    sol_json_scanner_init(&sub_scanner, slice.data, slice.len);
```

Comment:
maybe add a helper `sol_json_scanner_init_from_slice()`?  this looks useful (static inline in the header).